client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -52,54 +57,50 @@ const PhpMyAdminCard = ( { translate, siteId, token, loading, disabled } ) => {
 
 	return (
 		<Card className="phpmyadmin-card">
-			<div className="phpmyadmin-card__icon">
-				<MaterialIcon icon="dns" size={ 32 } />
-			</div>
-			<div>
-				<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
-				<p>
+			<MaterialIcon icon="dns" size={ 32 } />
+			<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
+			<p>
+				{ translate(
+					'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Manage your databases with phpMyAdmin and run a wide range of operations with MySQL.' )` *ES Score: 10*

Reply:
There is only one database, so we've adjusted the copy to better reflect that.